### ISSUE TITLE
Clean up the Container hierarchy

### ISSUE THREAD

**Issue by kosack** [cta-observatory/ctapipe#500]
We need to look in detail at all the data that is currently in the Container hierarchy, as well as data that needs to be added (for things like Tracking, Pointing, HV monitoring, etc.) and decide if:

* the structure is consistent or needs updating
* the use-cases (e.g. how data are read or written into them) are met or understood
* data items are missing or redundant

Note that the idea for at least some Containers is that they are schemas for output or input files (and can thus be serialized to HDF5, FITS, or database tables).  So making them fit into that context is important.

**Comment by MaxNoe** [comment]
See also my proposal for typed fields: #501

**Comment by kosack** [comment]
Yes, perhaps we should wait until the typed fields are implemented, and then we can re-evaluate the contents.

**Comment by kosack** [comment]
It was decided in Annecy that we will change the container hierarchy such that the *event* containers only contain things that change per-event, and there will be a separate *header* (better name hopefully) container that has the instrument and other observation-specific information inside it.
